src/main/java/uk/gov/hmcts/reform/unspec/model/Address.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.unspec.model;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonNaming(PropertyNamingStrategy.UpperCamelCaseStrategy.class)
```

Comment:
Is this required? I thought this is default naming strategy.

Reply:
CCD field names have first letter Uppercase